endoreg_db/views/anonymization/overview.py

```
@@ -242,4 +203,14 @@ def clear_processing_locks(request):
         return Response(
             {"error": "Failed to clear locks"}, 
             status=status.HTTP_500_INTERNAL_SERVER_ERROR
-        )
\ No newline at end of file
+        )
+        
+@api_view(['GET'])
+@permission_classes(DEBUG_PERMISSIONS)
+def has_raw_video_file(request, file_id):
```

Ruff (0.14.1)

Unused function argument: `request`

(ARG001)